packages/web-components/src/components/va-alert-sign-in/va-alert-sign-in.tsx

```diff
@@ -0,0 +1,310 @@
+import {
+  Component,
+  Element,
+  Event,
+  EventEmitter,
+  Host,
+  Prop,
+  h,
+} from '@stencil/core';
+import classnames from 'classnames';
+import { VariantNames } from './VariantNames';
+
+/**
+ * @componentName Alert - Sign In
+ * @maturityCategory caution
+ * @maturityLevel candidate
+ */
+@Component({
+  tag: 'va-alert-sign-in',
+  styleUrl: 'va-alert-sign-in.scss',
+  shadow: true,
+})
+export class VaAlertSignIn {
+  @Element() el!: any;
+
+  /**
+   * Determines the text content and border/background color.
+   */
+  @Prop() variant: VariantNames = VariantNames.signInRequired;
+
+  /**
+   * If `true`, doesn't fire the CustomEvent which can be used for analytics tracking.
+   */
+  @Prop() disableAnalytics?: boolean = false;
+
+  /**
+   * If `true`, the alert will be visible.
+   */
+  @Prop() visible?: boolean = true;
+
+  /**
+   * For the 'optional' variant, how long the respondent has to submit their form
+   */
+  @Prop() timeLimit?: string = '15 minutes';
+
+  /**
+   * For the 'optional' variant, the link to the form to complete without signing in
+   */
+  @Prop() noSignInLink?: string;
+
+  /**
+   * Fires when the component has successfully finished rendering for the first
+   * time.
+   */
+  @Event({
+    eventName: 'va-component-did-load',
+    composed: true,
+    bubbles: true,
```

Comment:
I'm not sure I understand why this is needed. Is there some authentication logic that is relying on this? 

Also would it be clearer if the custom event name mentioned that this is the va-alert-sign-in component that loaded? ie `va-alert-sign-in-did-load`

Reply:
This was leftover from the original va-alert code, I'm not sure that it's completely necessary either. I'll remove it.